src/db-service/Users/queries.ts

```diff
@@ -12,3 +12,13 @@ query allUsersBirthdays {
     }
   }  
 `;
+
+export const getLoggedInUser = gql`
+query getLoggedInUser($userName: String!, $password: String!) {
+  loggedInUser : allUsers(condition: {userName: $userName, password: $password}) {
+    nodes {
```

Comment:
למה לא לשלוף גם את השם? למקרה ונרצה לשמור אותו בעתיד להצגה

Reply:
It’s not necessary right now, and we are not sure it will be displayed

Reply:
We need the name for the news page with all the comments and posts//